src/extension.ts

```diff
@@ -41,4 +54,28 @@ export function activate(context: ExtensionContext) {
     
   });
 
+  function getSettings(): Settings {
+   
+    let configXML = workspace.getConfiguration('xml');
+    
+    let configCatalogs = configXML.get('catalogs') as String[];
+    
+    let configFormats = configXML.get('format');    
+    configFormats["indentUsingSpaces"] = workspace.getConfiguration("editor").get("insertSpaces");
+    configFormats["tabSize"] = workspace.getConfiguration("editor").get("tabSize");
+
+    let configLogs = configXML.get('logs');
+    if(configLogs['file'] == null) {
+      configLogs['file'] = logfile;
+    }
+
+    let settings: Settings = {
+      catalogs: configCatalogs,
+      logs: configLogs,
+      format: configFormats
+    }
+
+    return settings;
```

Comment:
doesn't the server expect all those settings to be prefixed with `lsp4xml.`?

Reply:
So I updated [the wiki](https://github.com/angelozerr/lsp4xml/wiki/Configuration) and tried to match the current layout of it and it seems like this is the nicest way to do it with the prefix being 'format'